src/filesystem/impls/filer/lib/HTMLRewriter.js

```diff
@@ -103,6 +104,35 @@ define(function (require, exports, module) {
         Array.prototype.forEach.call(elements, rewritePath);
     };
 
+    HTMLRewriter.prototype.parseScripts = function() {
+         var elements = this.doc.querySelectorAll('script');
+        var dir = this.dir;
+
+        function rewritePath(element) {
+            // Skip any links for protocols (we only want relative paths)
+            var path = element.getAttribute('src');
+            if(!Content.isRelativeURL(path)) {
+                if(element.getAttribute('data-brackets-id') && !jsEnabled) {
+                    element.type = "text/-scripts-disabled";
```

Comment:
Should we do `setAttribute('type', 'text/x-scripts-disabled')` here instead of touching `.type` directly? Probably?